### ISSUE TITLE
Implementar UI mobile de "Instruções"

### ISSUE THREAD

**Issue by tuliooassis** [multei/web#270]

**Comment by jimmyandrade** [comment]
@tuliooassis depois vamos refinar esse?

**Comment by tuliooassis** [comment]
@jimmyandrade claro! Essa issue é sobre a personalização da página de Instruções. Acredito que você tinha comentado algo sobre consultar alguém de UX.

**Comment by jimmyandrade** [comment]
@tuliooassis é verdade 😄 Essa semana pretendo buscar por essa pessoa, inclusive.
